hoomd/md/force.py

```diff
@@ -117,212 +118,145 @@ def additional_virial(self):
         return numpy.array(virial, dtype=numpy.float64)
 
 
-class constant(Force):  # noqa - this will be renamed when it is ported to v3
-    R"""Constant force.
+class Custom(Force):
+    """Custom forces implemented in python.
 
-    Args:
-        fvec (tuple): force vector :math:`[force]`
-        tvec (tuple): torque vector :math:`[force \cdot length]`
-        fx (float): x component of force, retained for backwards compatibility
-          :math:`[\mathrm{force}]`
-        fy (float): y component of force, retained for backwards compatibility
-          :math:`[\mathrm{force}]`
-        fz (float): z component of force, retained for backwards compatibility
-          :math:`[\mathrm{force}]`
-        group (``hoomd.group``): Group for which the force will be set.
-        callback (`callable`): A python callback invoked every time the forces
-            are computed
-
-    :py:class:`constant` specifies that a constant force should be added to
-    every particle in the simulation or optionally to all particles in a group.
+    This class serves as a base class for all custom forces and should not be
+    instantiated directly. A custom force should be derived from this class, and
+    should override a method called `set_forces` in order to set the force data
+    however they wish. Custom forces have direct, zero-copy access to
+    HOOMD-Blue's underlying force buffers via either the
+    `cpu_local_force_arrays` or `gpu_local_force_arrays` property. Choose the
+    property that corresponds to the device you wish to alter the data on. In
+    addition to zero-copy access to force buffers, custom forces have access to
+    the local snapshot API via the ``_state.cpu_local_snapshot`` or the
+    ``_state.gpu_local_snapshot`` property. See the documentation in
+    ``hoomd.state`` for more information on the local snapshot API.
```

Comment:
```suggestion
    Derive a custom force class from `Custom`, and override the `set_forces` method to  compute forces on particles. Use have direct, zero-copy access to the C++ managed buffers via either the
    `cpu_local_force_arrays` or `gpu_local_force_arrays` property. Choose the
    property that corresponds to the device you wish to alter the data on. In
    addition to zero-copy access to force buffers, custom forces have access to
    the local snapshot API via the ``_state.cpu_local_snapshot`` or the
    ``_state.gpu_local_snapshot`` property. 
    
    See Also:
      See the documentation in `hoomd.State` for more information on the local snapshot API.
```
This will need to be rewrapped and check that the `See Also:` syntax is correct to create a see also info block in the generated docs.